sketch/tensor_full.hpp

```diff
@@ -0,0 +1,101 @@
+#pragma once
+
+#include "immintrin.h" // for AVX
+#include "nmmintrin.h" // for SSE4.2
+#include "sequence/alphabets.hpp"
+#include "sketch//sketch_base.hpp"
+#include "util/multivec.hpp"
+#include "util/timer.hpp"
+#include "util/utils.hpp"
+
+#include <cassert>
+#include <cmath>
+#include <iostream>
+#include <random>
+
+namespace ts { // ts = Tensor Sketch
+
+/**
+ * Computes tensor sketches for a given sequence as described in
+ * https://www.biorxiv.org/content/10.1101/2020.11.13.381814v1
```

Comment:
Is this copied over from another class? :) I think this class doesn't compute sketches.